src/pybind/mgr/dashboard/services/cluster.py

```diff
@@ -44,6 +47,13 @@ def from_db(cls):
     @classmethod
     def get_capacity(cls) -> ClusterCapacity:
         df = mgr.get('df')
+        total_objects = 0
+        total_pool_bytes_used = 0
+        for pool in df['pools']:
+            total_objects = total_objects + pool['stats']['objects']
+            total_pool_bytes_used = total_pool_bytes_used + pool['stats']['bytes_used']
```

Comment:
we should review if this is correct with rgw folks.

Reply:
Yeah I will confirm this with them